### DISCUSSION TITLE
Hint: toggling power to bluetooth controller helps get stable connection

### DISCUSSION THREAD

**Discussion by rdstevens** [fl4p/batmon-ha#91] [General]
When reconfiguring batmon-ha, I often end up failing to get a connection to the battery BMSes. Using buetoothctl on the terminal and doing power off/power on usually fixes the problem.

Might be worth noting in the docs - or perhaps building in to the code to power-cycle (as an option) when restarting the service.

**Comment by fl4p** [comment]
Thanks, added this to the readme. And I'll add a new option to power cycle bluetooth on startup
